SteamList/Controllers/NewsViewController.swift

```diff
@@ -40,4 +40,18 @@ extension NewsViewController: UITableViewDataSource {
         cell.configCell(with: news)
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let news = newsList[indexPath.row]
```

Comment:
There is no need to create another instance. It could be done in just on line like:
`navigateToNewsDetails(with: newsList[indexPath.row])`

Reply:
done